cli/app.go

```diff
@@ -243,6 +245,8 @@ func (c *App) setup(app *kingpin.Application) {
 	app.Flag("advanced-commands", "Enable advanced (and potentially dangerous) commands.").Hidden().Envar(c.EnvName("KOPIA_ADVANCED_COMMANDS")).StringVar(&c.AdvancedCommands)
 	app.Flag("track-releasable", "Enable tracking of releasable resources.").Hidden().Envar(c.EnvName("KOPIA_TRACK_RELEASABLE")).StringsVar(&c.trackReleasable)
 	app.Flag("dump-allocator-stats", "Dump allocator stats at the end of execution.").Hidden().Envar(c.EnvName("KOPIA_DUMP_ALLOCATOR_STATS")).BoolVar(&c.dumpAllocatorStats)
+	app.Flag("upgrade-owner-id", "Repository format upgrade owner-id.").Hidden().Envar(c.EnvName("KOPIA_REPO_UPGRADE_OWNER_ID")).StringVar(&c.upgradeOwnerID)
```

Comment:
As discussed offline: let's use the repo (maintenance) owner, instead of introducing another entity here. This means, the upgrade needs to be done from the repo owner instance. It has the advantage that 1. ensures maintenance is not running concurrently, and 2. it allows delayed execution (as part of maintenance) if we decide to do so in the future.

Reply:
Discussed with @julio-lopez , will make this change in a separate PR.